tool/jt.rb

```diff
@@ -1574,7 +1574,7 @@ def install_graal_core
           java_home = Dir["#{dir}/jdk1.8.0*"].sort.first
         end
       elsif MAC
-        jvmci_version = "jvmci-0.24"
+        jvmci_version = "jvmci-0.29"
```

Comment:
`0.29`? The version number is unrelated to GraalVM now - 0.25 of JVMI came out today.